packages/pwa-kit-dev/src/configs/webpack/config.js

```diff
@@ -403,12 +403,7 @@ const enableReactRefresh = (config) => {
             new ReactRefreshWebpackPlugin({
                 overlay: false
             })
-        ],
-        output: {
-            ...config.output,
-            // Setting this so that *.hot-update.json requests are resolving
-            publicPath: '/mobify/bundle/development/'
-        }
```

Comment:
This is one of the notable changes in this PR. Rather than hard coding `/mobify/bundle/development` for HMR, the publicPath is now dynamically being set in the SDK's main.jsx.